src/admin-views/settings/emails/message-templates/list-recipient.php

```diff
@@ -1,22 +1,19 @@
 <?php
 /**
- * Tickets Emails Message Template List Recipient
+ * Tickets Emails List Recipient
  *
- * @since  5.5.6   Recipient for list item of email message templates for Emails settings tab.
+ * @since  5.5.6   Recipient for list item of email templates for Emails settings tab.
  *
- * @var Array[]  $templates  Array of template info.
- * @var Array    $template   Template info.
+ * @var Email_Abstract[]  $emails  Array of email info.
+ * @var Email_Abstract    $email   Email info.
  */
 
-// @todo $templates variable will be an array of Message_Template objects in the future.
-// @todo $template variable will be a Message_Template object in the future.
-
-// If no template, bail.
-if ( empty( $template ) ) {
+// If no email, bail.
+if ( empty( $email ) ) {
 	return;
 }
 
 ?>
 <div class="tec-tickets__admin-settings-emails-template-list-item-recipient">
-	To: <?php echo esc_html( $template['recipient'] ); ?>
+	<?php echo esc_html__( 'To:', 'event-tickets' ); ?> <?php echo esc_html( $email->recipient ); ?>
```

Comment:
```suggestion
	<?php echo sprintf( 
			// Translators: %s: The email recipient.
			esc_html__( 'To: %s', 'tribe-common' ), 
			esc_html( $email->recipient )
		); ?>
```

Normally, this type of translations need to be managed with placeholders. We don't know if, in other languages, the `To:` goes after, and so on. It's not always the case, but if we have the chance I think it's worth it to do it.